src/west/app/project.py

```diff
@@ -932,6 +940,10 @@ def update(self, project):
                 stats['checkout new manifest-rev'] = perf_counter() - start
             _post_checkout_help(project, current_branch, sha, is_ancestor)
 
+        # Check whether project has any submodules and if it does update them.
+        if project.submodules:
+            update_submodules(project.submodules, self.topdir + '/' + project.path)
```

Comment:
Use `os.path.join(self.topdir, project.path)` for more portable code (unless west doesn't care)

Reply:
Good idea, done.